src/test/Components/ShareImageModal/ShareImageModal.test.js

```diff
@@ -101,107 +99,24 @@ describe('Create Share To Regions Modal', () => {
   });
 
   test('select options disabled correctly based on status and region', async () => {
-    renderCustomRoutesWithReduxRouter(`share/${composeId}`, mockState, routes);
+    renderCustomRoutesWithReduxRouter(`share/${composeId}`, {}, routes);
 
-    const selectToggle = screen.getByRole('button', { name: /options menu/i });
+    const selectToggle = await screen.findByRole('button', {
+      name: /options menu/i,
+    });
     // eslint-disable-next-line testing-library/no-unnecessary-act
```

Comment:
We can get rid of most (maybe all) `// eslint-disable-next-line testing-library/no-unnecessary-act` lint, but that can be done in a separate PR.